teerex/src/migrations.rs

```diff
@@ -37,40 +34,50 @@ mod v0 {
 pub mod v1 {
 	use super::*;
 
+	#[derive(
+		Encode, Decode, Default, Copy, Clone, PartialEq, Eq, sp_core::RuntimeDebug, TypeInfo,
+	)]
+	pub struct SgxTcbInfoOnChainV1 {
+		pub issue_date: u64,
+		pub next_update: u64,
+	}
+
+	#[storage_alias]
+	pub type SgxTcbInfo<T: Config> =
+		StorageMap<Pallet<T>, Blake2_128Concat, Fmspc, SgxTcbInfoOnChainV1, OptionQuery>;
+
 	pub struct MigrateV0toV1<T>(sp_std::marker::PhantomData<T>);
 
 	impl<T: Config + frame_system::Config> OnRuntimeUpgrade for MigrateV0toV1<T> {
 		#[cfg(feature = "try-runtime")]
 		fn pre_upgrade() -> Result<Vec<u8>, &'static str> {
-			let current_version = Pallet::<T>::current_storage_version();
 			let onchain_version = Pallet::<T>::on_chain_storage_version();
-			ensure!(onchain_version == 0 && current_version == 1, "only migration from v0 to v1");
 
 			let enclave_count = v0::EnclaveCount::<T>::get() as u64;
-			log::info!(target: TARGET, "{} enclaves will be purged", enclave_count,);
+			log::info!(
+				"teerexV1: {} v0 enclaves are present before eventual upgrade",
```

Comment:
the log target gets swallowed by try-runtime. therefore using prefix in log msg

Reply:
How do they do it in substrate? Like this, it will probably be displayed in try-runtime, but actually not be logged in production.